selftests/deployment/README.rst

```diff
@@ -0,0 +1,105 @@
+README
+======
+
+Here is a basic playbook (`deployment.yml`) to test the deployment of
+Avocado in a number of different environments, and installed via
+different methods.
+
+This code is intended to be used as regular playbook, so roughly the
+following steps should be followed:
+
+ 1) Install ansible. If you need help, refer to
+    https://docs.ansible.com/ansible/latest/installation_guide
+
+ 2) Adjust your inventory, that is, the hosts that will be used on the
+    playbook execution. The simplest way is to edit the `inventory`
+    file this directory.
+
+ 3) (OPTIONAL) adjust your variables on `vars.yml` or change the -e option via
+    command-line. Example::
+
+    $ ansible-playbook .... -e "method=pip"
+ 
+ 4) Run a playbook with::
+
+    $ ansible-playbook -i inventory -c local deployment.yml -e "method=pip"
+
+.. note:: Note that `-e` option it is not mandatory.
+
+Main variables
+==============
+
+Besides the basic variables (visit `vars.yml`) we have two main variables to
+control the execution model: `method` and `avocado_vt`.
+
+method
+------
+
+ 1. `pip`: Install Avocado and Plugins using pip on a Python virtual
+ environment (this is the default method).
+
+ 2. `copr`: Install Avocado and Plugins using RPM, from a Copr
+ repository;
+
+ 3. `official`: Install Avocado and Plugins using RPM, from the official
```

Comment:
And as expected the same thing happens with `method=official`:

```
TASK [avocado : Install Avocado plugins available on Fedora only] ********************************************************************************
failed: [localhost] (item=python3-avocado-plugins-runner-remote) => {"ansible_loop_var": "item", "changed": false, "failures": ["No package python3-avocado-plugins-runner-remote available."], "item": "python3-avocado-plugins-runner-remote", "msg": "Failed to install some of the specified packages", "rc": 1, "results": []}
failed: [localhost] (item=python3-avocado-plugins-runner-vm) => {"ansible_loop_var": "item", "changed": false, "failures": ["No package python3-avocado-plugins-runner-vm available."], "item": "python3-avocado-plugins-runner-vm", "msg": "Failed to install some of the specified packages", "rc": 1, "results": []}
```

Reply:
Agree. Removed on v3.